File: 2503-longest-subarray-with-maximum-bitwise-and/longest-subarray-with-maximum-bitwise-and.java
class Solution {
    public int longestSubarray(int[] nums) {
        int count=0, largest=0, result=0;
        for (int i: nums){
            if(i>largest){
                largest = i;
                count = 1;
                result = 0;
            }else if(i == largest){
                count++;
            }else{
                result = (count>result)?count:result;
                count = 0;
            }
        }
        result = (count>result)?count:result;
        return result;
    }
}

File: 2503-longest-subarray-with-maximum-bitwise-and/longest-subarray-with-maximum-bitwise-and.py
class Solution(object):
    def longestSubarray(self, nums):
        count = 0
        largest = 0
        result = 0
        for i in nums:
            if i>largest:
                largest = i
                count = 1
                result = 0
            elif i == largest:
                count += 1
            else:
                result = count if count>result else result
                count = 0
        result = count if count>result else result
        return result
        